scripts/install_third_party_libs.py

```diff
@@ -69,6 +69,25 @@
     common.OPPIA_TOOLS_DIR, 'pylint-quotes-0.1.8', 'configparser.py')
 
 
+def tweak_yarn_executable():
+    """When run yarn on Windows, the file `yarn` will be excecuted (
```

Comment:
Spelling: executed

More generally, though, I had to read this several times before I understood it. Here's a suggested revision (but please check it for correctness, I'm not 100% sure about it):

"When yarn is run on Windows, the file `yarn` will be executed by default. However, this file is a bash script, and can't be executed directly on Windows. So, to prevent Windows automatically executing it by default (while preserving the behaviour on other systems), we rename it to `yarn.sh` here."

Reply:
Updated